shopBuddy/shopBuddy/SpecificListViewController.m

```diff
@@ -0,0 +1,66 @@
+//
+//  ShoppingListViewController.m
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/6/22.
+//
+
+#import "SpecificListViewController.h"
+#import "ListItemCell.h"
+#import "Item.h"
+#import "UIImageView+AFNetworking.h"
+#import "ShoppingList+Persistent.h"
+#import "Item+Persistent.h"
+
+@interface SpecificListViewController () <UITableViewDataSource, UITableViewDelegate>
+{
+    __weak IBOutlet UITableView *tableView;
+    NSArray<Item *> *items;
+    
+}
+@property (weak, nonatomic) IBOutlet UILabel *list_label;
+
+@end
+
+@implementation SpecificListViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    tableView.dataSource = self;
+    tableView.delegate = self;
+    tableView.rowHeight = UITableViewAutomaticDimension;
+    _list_label.text = [NSString stringWithFormat:@"%@ Shopping List", self.list.store_name];
+    [self.list fetchItemsInList:^(NSArray<Item *> *items, NSError *error) {
```

Comment:
You shouldn't need this because `_list` should already have this (i.e. item list).